src/pages/projects/create/Create.tsx

```diff
@@ -144,7 +144,6 @@ export default function Create() {
               clearButton={Boolean(project?.client_id)}
               onClearButtonClick={() => handleChange('client_id', '')}
               errorMessage={errors?.errors.client_id}
-              staleTime={Infinity}
```

Comment:
@beganovich @turbo124 Setting staleTime to Infinity manually for the Create Project page is no longer necessary because we have already changed it to be set by default.